lang/fr.json

```diff
@@ -409,14 +410,14 @@
   "createPledge.message": "Un message pour la communauté (optionnel)",
   "createPledge.monthly": "Mensuel",
   "createPledge.name": "Nom",
-  "createPledge.onceTheyCreateIt": "Une fois le collectif créé (et que nous avons vérifié qu'ils possèdent l’URL mentionnée), vous recevrez un email vous demandant de compléter votre promesse de don.",
+  "createPledge.onceTheyCreateIt": "Once they create it (and verify that they own the URL you’ll enter in this form), you will receive an\n                email to ask you to fulfill your pledge.",
   "createPledge.oneTime": "Ponctuelle",
   "createPledge.pledgeAs": "Faire la promesse en tant que :",
   "createPledge.pledgeDetails": "Détails :",
-  "createPledge.priviledge": "Vous avez le privilège de nommer et de décrire ce fantastique projet. Nous lui créerons une\n                        page dédiée pour que d'autres personnes puissent le trouver et accompagner votre promesse de don.",
+  "createPledge.priviledge": "You’ve earned the priviledge to name and describe this awesome cause. We’ll create a pledged\n                        collective page for it so other people can find it and pledge to it too.",
   "createPledge.profile": "Choisissez un profil",
   "createPledge.signinToCreate": "<signin-link>Connectez-vous ou créez un compte</signin-link> pour effectuer une promesse de don.",
-  "createPledge.why": "Si le groupe ou le projet que vous souhaitez soutenir n'est pas encore sur Open Collective, vous pouvez faire une \n                promesse de don. Cela les motivera à créer un collectif transparent pour leurs activités et vous donnera \n                une plus grande visibilité sur comment votre argent est dépensé pour faire avancer cette mission.",
+  "createPledge.why": "If the cause or collective that you want to support is not yet on Open Collective, you can make a\n                pledge. This will incentivize them to create an open collective for their activities and offer you much\n                more visibility on how your money is spent to advance their cause.",
```

Comment:
Should stay in French.

Reply:
My goodness! I ran `npm run langs:update` multiple times before I made a commit 😟

Reply:
Sorry, it's surely related with the CRLF issue.